.gitignore

```diff
@@ -18,3 +18,6 @@
 
 .byebug_history
 .idea
+
+/coverage.data
```

Comment:
replace these 2 lines with `coverage` it should take care of excluding the entire folder